NEWS

```diff
@@ -122,6 +124,11 @@ CHANGES WITH 239 in spe:
           search-binaries-default'. It's generally recommended to continue to
           use absolute paths for all binaries specified in unit files.
 
+        * Units gained a new load state "bad-setting", which is used when a
+          unit file was loaded, but contained fatal errors which prevent it
+          from being started (for example, an ExecStart= path which references
+          a non-existent executable).
```

Comment:
the example isn't right. This state is entered if a service unit has no ExecStart=/ExecStop= line at all, but whether an executable exists at all is only determined when we are going to start it. And that's a good thing as the binary might not exist during early boot yet because its file system is not mounted yet

Reply:
So I **did** test this, before committing. ExecStart=asfdasfasd fails (early), but ExecStart=/bin/fasdfasdfsdf "succeeds" (fails late). But you're right, the example is wrong as written.